tools/defaults.bzl

```diff
@@ -9,8 +9,8 @@ load("@npm_angular_bazel//:index.bzl", _ng_module = "ng_module", _ng_package = "
 load("@npm_bazel_typescript//:defs.bzl", _ts_library = "ts_library")
 
 DEFAULT_TSCONFIG = "//:tsconfig.json"
-NG_VERSION = "^7.0.0"
-RXJS_VERSION = "^6.0.0"
+NG_VERSION = "^8.0.0-beta"
```

Comment:
We also have to update `platformVersion` at https://github.com/ngrx/platform/blob/master/modules/schematics-core/utility/libs-version.ts#L1